src/commonMain/resources/db/migration/V30__Follows.sql

```diff
@@ -0,0 +1,27 @@
+CREATE SEQUENCE public."follows_followId_seq"
+    INCREMENT 1
+    START 1
+    MINVALUE 1
+    NO MAXVALUE
+    CACHE 1;
+
+ALTER TABLE public."follows_followId_seq" OWNER TO beatmaps;
+
+CREATE TABLE public.follows
```

Comment:
Need a unique index

Reply:
(Both to stop people following users twice and because right now checking if you follow a user requires scanning this whole table)